CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Change Log
 All notable changes to this project will be documented in this file. This change log follows the conventions of [keepachangelog.com](http://keepachangelog.com/).
 
+## [0.14.3 2019-11-4]
+## Added
+- default driver is 78.0.3904.70
```

Comment:
Default? you mean there's a way to change it from the default?

Reply:
Perhaps? I'd have to go through and look. This is actually just a continuation of the wording used in previous updates of this type.